rosa_backing_up_and_restoring_applications/backing-up-applications.adoc

```diff
@@ -15,28 +15,39 @@ This is a two stage process:
 
 include::modules/oadp-preparing-aws-credentials.adoc[leveloffset=+1]
 
-include::modules/oadp-installing-oadp-rosa-sts.adoc[leveloffset=+1]
+include::modules/installing-oadp-rosa-sts.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 [id="oadp-addtl-resources_{context}"]
 .Additional resources
 
 * xref:../rosa_backing_up_and_restoring_applications/backing-up-applications.adoc#oadp-preparing-aws-credentials_rosa-backing-up-applications[Preparing AWS credentials]
 
+//Added the following two modules to make a more complete guide.
+
+[id="oadp-rosa-backing-up-and-cleaning-example"]
+== Backing up workloads on OADP with ROSA STS
+
+include::modules/performing-a-backup-oadp-rosa-sts.adoc[leveloffset=+2]
+
+include::modules/cleanup-a-backup-oadp-rosa-sts.adoc[leveloffset=+2]
+
 [id="rosa-backing-up-applications-known-issues"]
 == Known issues
-.Restic, Kopia, and DataMover are not supported or recommended
 
+* Restic, Kopia, and DataMover are not supported or recommended
```

Comment:
This bullet definitely needs a period, as it is a complete sentence. The other bullets should get a period also, for consistency, even though it might look odd, as they are clearly not a complete sentence.